### PR TITLE
mysql: MyDecimal add more methods and tests.

### PR DESCRIPTION
**Pull request by coocood**

### CODE REVIEW

**Comment by coocood** [comment]
@shenli @tiancaiamao @zimulala PTAL

File: mysql/mydecimal.go [hunk 1]
```diff
@@ -133,6 +134,140 @@ func (d *MyDecimal) String() string {
...
+	}
+	if digitsInt > 0 {
+		digitsInt -= countLeadingZeroes((digitsInt-1)%digitsPerWord, d.wordBuf[wordIdx])
+	} else {
+		digitsInt = 0
+	}
+	return
+}
+
+// ToString convert decimal to its printable string representation.
```
Comment (zimulala) [inline mysql/mydecimal.go]: convert -> converts

**Comment by shenli** [comment]
LGTM

**Comment by tiancaiamao** [comment]
LGTM

**Pull request merged**